dags/main_summary_backfill.py

```diff
@@ -18,8 +18,8 @@
 
 t0 = EMRSparkOperator(task_id="main_summary_backfill",
                       job_name="Main Summary Backfill",
-                      execution_timeout=timedelta(hours=10),
-                      instance_count=20,
+                      execution_timeout=timedelta(hours=24),
```

Comment:
I ran one day and it took a bit under 3 hours. I could bump it up higher for safety. Consider that the current main_summary only uses 10 nodes